homeassistant/components/tibber/sensor.py

```diff
@@ -366,6 +421,7 @@ async def async_callback(self, payload):
         if live_measurement is None:
             return
 
+        timestamp = datetime.fromisoformat(live_measurement.pop("timestamp"))
```

Comment:
Is this UTC time?

Reply:
No, timezone of the home

Reply:
Ok, then we need to convert it to UTC time.

Reply:
Ok, Can I ask why?
Since this also shows up in the frontend, I think local time (with timezone info) would make more sense.

Reply:
All times in the state machine must be UTC time. The frontend is responsible for converting to local time when needed.

Reply:
Fixed here: https://github.com/home-assistant/core/pull/50951/files#diff-2a4673e367cb32db74b949cd9c36f5ad31766f0147dc2673910879daa64b85d4R390
Need to replace hour before converting to utc